### ISSUE TITLE
Commands not working

### ISSUE THREAD

**Issue by chumpydo** [RockinChaos/ItemJoin#117]
Sorry to bother you again <3

It seems when I add a command and respective information to an item, the plugin stops working. It still is enabled, it's just the items don't load. None of them do. 

Item YML: https://hastebin.com/ugenopuyaf.http

**Comment by RockinChaos** [comment]
Resolved; https://pastebin.com/yt7NpZSg

You cannot use apostrophes ( ' ) in a .yml file unless it is putting text inside it. The error occurs here; Lightyear's so I changed it to Lightyears. Standard java YAML arguments, out of my control to change sorry.

Btw It shows what went wrong in the console if you go through the startup or /itemjoin reload. I will show up instantly. It shows the location of the error. 

http://i.imgur.com/Fo0KvRU.png
